<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_DatabaseConnection</class>
 <widget class="QDialog" name="Dialog_DatabaseConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>258</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>258</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect to Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_DialogDatabaseConnection_ServerPortNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_DialogDatabaseConnection_ServerPortNumber">
       <property name="text">
        <string>Server Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_DialogDatabaseConnection_ServerName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_DialogDatabaseConnection_BrowserPortNumber">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_DialogDatabaseConnection_ServerName">
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_DialogDatabaseConnection_BrowserPortNumber">
       <property name="text">
        <string>Browser Port</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_DialogDatabaseConnection_Warning">
       <property name="text">
        <string>Before you click on connect, make sure your database is open and activate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_DialogDatabaseConnection_Password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_DialogDatabaseConnection_Username"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_DialogDatabaseConnection_Password">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_DialogDatabaseConnection_Username">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_DialogDatabaseConnection_OpenSchema">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_DialogDatabaseConnection_OpenSchema">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load database schema</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox_DialogDatabaseConnection">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_DialogDatabaseConnection_Username</tabstop>
  <tabstop>lineEdit_DialogDatabaseConnection_Password</tabstop>
  <tabstop>lineEdit_DialogDatabaseConnection_ServerName</tabstop>
  <tabstop>lineEdit_DialogDatabaseConnection_ServerPortNumber</tabstop>
  <tabstop>lineEdit_DialogDatabaseConnection_BrowserPortNumber</tabstop>
  <tabstop>pushButton_DialogDatabaseConnection_OpenSchema</tabstop>
  <tabstop>lineEdit_DialogDatabaseConnection_OpenSchema</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
